### PR TITLE
add: column for schedulable in agent table and provide updateable UI for its value

### LINKED ISSUE #1166: UI to query and update Agent's schedulable field
**Is your feature request related to a problem? Please describe.**
기능 요청이 특정 문제에 연관된 것이라면 여기에 그 문제를 설명해주세요.
Since the information of whether each agent is schedulable or not has been provided, it needs to be exposed in webUI, likewise.
And the `schedulable` value affects the scheduling algorithm, therefore we should provide updating UI so that the user(superadmin only) could change the status manually.

**Describe the solution you'd like**
어떤 기능이 있으면 좋겠는지 자세히 설명해주세요.
1. Let's add a column for `schedulable` field into the agent table. so that user(Admin Role) can find out whether the agent server can create session(s) in there, or not.
2. Let's add updating UI for each agent's `schedulable` value.

**Describe alternatives you've considered**
혹시 다른 대안들을 생각해본 적이 있다면 함께 적어주세요.

**Additional context**
기능 요청에 대해 보다 잘 이해할 수 있는 다른 맥락을 기술해주세요.

### PR DESCRIPTION
**Pull request by lizable**
This PR resolves #1166.

> ⚠️&nbsp; NOTE: `schedulable` field column is displayed in the active tab only.

related PR: [backend.ai-manager#500](https://github.com/lablup/backend.ai-manager/pull/500)

### Before
-----
schedulable-field-before

### After
-----
schedulable-field-after

### CODE REVIEW

File: resources/i18n/en.json
Comment (adrysn) [inline resources/i18n/en.json]: Let's use shorter messages. Just "No changes" would be sufficient here.

File: src/components/backend-ai-agent-list.ts
Comment (adrysn) [inline src/components/backend-ai-agent-list.ts]: Indentation?

File: src/lib/backend.ai-client-node.ts
Comment (adrysn) [inline src/lib/backend.ai-client-node.ts]: Typo?

File: src/components/backend-ai-agent-list.ts
Comment (adrysn) [inline src/components/backend-ai-agent-list.ts]: `schedulable` mutation exists only for Manager>=21.09 so this query will raise an error on Manager<=21.03. Since Web-UI main branch currently supports multiple versions of Managers, I think the `schedulable` field should be version-checked. Version checking might be performed in the JS SDK library, `agent.list`.

**Review by adrysn — CHANGES_REQUESTED** [review]
I have some comments.

**Review by adrysn — CHANGES_REQUESTED** [review]
Just after the update, the "Schedulable" column is not updated. User needs to refresh the browser to see the column is updated.

**Review by adrysn — APPROVED** [review]
LGTM. I think this PR can be merged after the manager counter-part is merged.

**Review by inureyes — APPROVED** [review]
LGTM. Except conflict. (Will be resolved now)

**Pull request merged**
